src/style.css

```diff
@@ -1,3 +1,63 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  --color-blue: #00acdc;
+  --gap: 20px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: auto 0;
+}
+
+.nav__list {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  display: flex;
+  align-items: stretch;
```

Comment:
This is a default value for `align-items` property, no need to specify it explicitly.

Reply:
Yeah, it is not necessary